src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,207 @@
 package core.basesyntax;
 
-public class MyHashMap<K, V> implements MyMap<K, V> {
+import java.util.Objects;
+import java.lang.reflect.Field;
+
+public class MyHashMap<K, V> implements MyMap<K, V>, Cloneable {
+
+    static class Node<V, K>{
+        final int hash;
+        final K key;
+        V value;
+        MyHashMap.Node<K,V> next;
+
+        public Node(int hash, K key, V value, Node<K, V> next) {
+            this.hash = hash;
+            this.key = key;
+            this.value = value;
+            this.next = next;
+        }
+
+    }
+
+    private int totalSize = 16;
+    final static float loadFactor = 0.75f;
```

Comment:
Naming convention for constants is uppercase with words separated by underscores

https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html